spec.txt

```diff
@@ -5354,9 +5354,18 @@ well.  ([reStructuredText](http://docutils.sourceforge.net/rst.html)
 takes a different approach, requiring blank lines before lists
 even inside other list items.)
 
-In order to solve of unwanted lists in paragraphs with
-hard-wrapped numerals, we allow only lists starting with `1` to
-interrupt paragraphs.  Thus,
+Interrupting a paragraph with an ordered list introduces an parsing
+ambiguity. What looks like an ordered marker might instead be a
+a numeral at the end of a sentence, continuing rather than
+interrupting the paragraph:
+
+``` markdown
+The number of windows in my house is
+14.  The number of doors is 6.
```

Comment:
```suggestion
14. The number of doors is 6.
```

The extra space was, I believe, not needed to illustrate the problem.

Reply:
I copied this exactly as it is in the example that follows. If it is really better to remove the
extra space, we should do it for both.